layouts/_default/baseof.html

```diff
@@ -24,6 +24,11 @@
     <meta name="twitter:card" content="summary" />
 
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link 
+  href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined"
```

Comment:
Rather than load everything from Google Fonts, could you only include the icons that are actually used?

Also, could you make sure that the icons are free license?